packages/i18n/src/messages/commonMessages.ts

```diff
@@ -150,6 +150,11 @@ const commonMessages = defineMessages({
     id: "2/YsnJ",
     description: "Delete action",
   },
+  remove: {
+    defaultMessage: "Remove",
```

Comment:
👍 for adding this as a reusable string.
👎 for not replacing the existing instance of "Remove"!

https://github.com/GCTC-NTGC/gc-digital-talent/blob/11b9ab89eb684b28d6a0b7c7da3b0032f6dbe0e3/apps/web/src/pages/Pools/EditPoolPage/components/SkillTable.tsx#L89-L93

https://github.com/GCTC-NTGC/gc-digital-talent/blob/85f4aed2b3c5955ed237a21863ddadd325e312f2/apps/web/src/lang/fr.json#L10913-L10916

Reply:
Doh!
[reuse remove string](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9592/commits/558855c127d65e7e4b4dcb7791108b4ff454d1e6)